pkg/dispatcher/operator/operator.go

```diff
@@ -58,10 +60,9 @@ func (op *Operator) Initialize() {
 	op.informer = informers.NewSharedInformerFactoryWithOptions(op.vegaclientset, 30*time.Second, informers.WithNamespace("vega"))
 	runtime.Must(calculationscheme.AddToScheme(scheme.Scheme))
 
-	// TODO: password: Get from Secret
 	redisClient := redis.NewClient(&redis.Options{
 		Addr:     op.redisURL,
-		Password: "vega12345", // temp for testing
+		Password: op.redisPW,
```

Comment:
the whole redisClient initialization can be transfer in main. See my comment above